language/use-cases/applying-llms-to-data/bigquery_ml_llm.ipynb

```diff
@@ -0,0 +1,1087 @@
+    "              MODEL `{PROJECT_ID}.{DATASET_ID}.{LLM_MODEL_NAME}`,\n",
+    "              (\n",
+    "              SELECT\n",
+    "                CONCAT('{PROMPT}', body) AS prompt,\n",
+    "                body\n",
+    "              FROM\n",
+    "                `bigquery-public-data.bbc_news.fulltext`\n",
+    "              LIMIT\n",
+    "                5),\n",
+    "              STRUCT(1 AS temperature, 1024 AS max_output_tokens, 0.8 AS top_p, 40 AS top_k, TRUE AS flatten_json_output));\n",
+    "        \"\"\"\n",
+    "result = run_bq_query(sql)\n",
+    "result.to_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "rSP17UbCYDR_"
+   },
+   "source": [
+    "This example extracts the brand names from product descriptions stored in the BigQuery thelook_ecommerce public dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i_Go_5XMSei7"
+   },
+   "outputs": [],
+   "source": [
+    "PROMPT = \"Please return the brand name listed in this product description. Here is an example. Product: TYR Sport Mens Solid Durafast Jammer Swim Suit, Brand: TYR ; Product: \"\n",
+    "\n",
+    "sql = f\"\"\"\n",
+    "          SELECT\n",
+    "            name AS product_description,\n",
+    "            CONCAT('{PROMPT}', 'product_description,',' Brand: ') as prompt_template,\n",
+    "            ml_generate_text_llm_result as llm_result\n",
+    "          FROM\n",
+    "            ML.GENERATE_TEXT(\n",
+    "              MODEL `{PROJECT_ID}.{DATASET_ID}.{LLM_MODEL_NAME}`,\n",
+    "              (\n",
+    "              SELECT\n",
+    "                CONCAT('{PROMPT}', name,' Brand: ') AS prompt,\n",
+    "                name\n",
+    "              FROM\n",
+    "                `bigquery-public-data.thelook_ecommerce.products`\n",
+    "              LIMIT\n",
+    "                5),\n",
+    "              STRUCT(1 AS temperature, 1024 AS max_output_tokens, 0.8 AS top_p, 40 AS top_k, TRUE AS flatten_json_output));\n",
+    "        \"\"\"\n",
+    "result = run_bq_query(sql)\n",
+    "result.to_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5JzOqTnIizNw"
+   },
+   "source": [
+    "#### Sentiment Analysis\n",
+    "\n",
+    "This example runs sentiment analysis on movie reviews stored in the BigQuery IMDB public dataset to determine whether the movie is positive, negative or neutral."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "4ALWl_i4oF-X"
+   },
+   "outputs": [],
+   "source": [
+    "PROMPT = \"Please categorize this movie review as either Positive, Negative or Neutral. Here is an example. Review: I dislike this movie, Sentiment: Negative\"\n",
+    "\n",
+    "sql = f\"\"\"\n",
+    "          SELECT\n",
+    "            review,\n",
+    "            CONCAT('{PROMPT}',' Review: review Category:') as prompt_template,\n",
```

Comment:
Probably want to change "Category:" to "Sentiment:" in this prompt or change the example from "Sentiment: Negative" to "Category: Negative"